### ISSUE TITLE
Forever stuck at loading 1 new model

### ISSUE THREAD

**Issue by Pareash** [lllyasviel/Fooocus#1903]
**Read Troubleshoot**

[x] I admit that I have read the [Troubleshoot](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) before making this issue.

I have similar issue it take forever to load and end up having errored message.
I have a AMD Ryzen 5 4600H with Radeon Graphics (512 MB) 3.00 GHz CPU, 8 GB Ram, and an NVIDIA GeForce GTX 1650.
Os: Windows 11

Please help i am trying since 2 weeks. Not single image generated. It always stuck at this stage.
I tried Systam swap but still not working.

Anything wrong with the hardware?

Prompt was simple: Cat

D:\Ai Tools\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.1.862
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 4096 MB, total RAM 7541 MB
Trying to enable lowvram mode because your GPU seems to have 4GB or less. If you don't want this use: --always-normal-vram
Set vram state to: LOW_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce GTX 1650 : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: D:\Ai Tools\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [D:\Ai Tools\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors].
Loaded LoRA [D:\Ai Tools\Fooocus_win64_2-1-831\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [D:\Ai Tools\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_version6Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cpu, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 8055833477153417220
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] cat, detailed intricate, serious, beautiful, highly detail, dramatic light, sharp focus, epic scenic full background, appealing, pretty, open, loving, generous, cool, divine, professional, artistic, passionate, inspiring, confident, inspired, vibrant, illuminated, transparent, magic, glowing, vivid, wonderful, symmetry, innocent, pure, coherent, cute, perfect
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] cat, sharp focus, cinematic, highly detailed, intricate, elegant, complex, exquisite, full color, cool dynamic colors, radiant light, professional background, enhanced quality, determined, dramatic, epic, stunning, attractive, caring, generous, noble, best, beautiful, futuristic, thought, perfect, majestic, thoughtful, awesome, illuminated, vibrant, shining, colorful, shiny
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 24.06 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model

D:\Ai Tools\Fooocus_win64_2-1-831>pause
Press any key to continue . . .

**Comment by mashb1t** [comment]
Relates to https://github.com/lllyasviel/Fooocus/issues/1471 (https://github.com/lllyasviel/Fooocus/issues/1471#issuecomment-1889734386), answer in https://github.com/lllyasviel/Fooocus/issues/1471#issuecomment-1889826961
